packages/components/src/alignment-control/README.md

```diff
@@ -0,0 +1,18 @@
+# AlignmentControl
```

Comment:
Should we call this `AlignmentMatrix` to better separate from `BlockAlignment`, etc?

Reply:
We could! Although I think having `Control` in the component name would be helpful, as it has the `value/onChange` interface.

`AlignmentMatrixControl`? or `AlignmentGridControl`?